### ISSUE TITLE
array_merge instead of plus symbol PHP7.0

### ISSUE THREAD

**Issue by purpleoar** [backdrop-contrib/globalredirect#4]
```
--- globalredirect/globalredirect.module	2015-04-22 03:34:08.000000000 +1200
+++ globalredirect/globalredirect.module	2018-04-18 21:49:18.562382736 +1200
@@ -378,7 +378,12 @@
     return $defaults;
   }
 
-  return config_get('globalredirect.settings', 'globalredirect_settings') + $defaults;
+  $temp = config_get('globalredirect.settings', 'globalredirect_settings');
+  if (is_array($temp)) {
+    return array_merge($temp, $defaults);
+  } else {
+    return $defaults;
+  }
 }
```
I tried making a pull request but failed! Found this converting a D7 site to backdrop (first one).

**Comment by prevoj** [comment]
I ran into the same problem. I upgraded a site from D7 to Backdrop and encountered the same issue as soon as I installed this module. The problem appears to be that initially there is no settings file so config_get returns NULL. Once the settings file is created, everything is good. I fixed my copy of the module almost the same way as above. I used is_set instead of is_array but still used the + operator like the original code and all was good.
